README.md

```diff
@@ -53,6 +53,20 @@ shell nor with `TMPDIR` being set. If database migrations fail with an extended
 sqlite error code 6410, pass `TMPDIR` pointing to a location, sqlite can write
 to.
 
+### Run with docker-compose
+```
+version: '3.3'
+services:
+  wastebin:
+    environment:
+      - WASTEBIN_DATABASE_PATH=/data/state.db
+    ports:
+      - "8088:8088"
```

Comment:
I would suggest to replace this with "127.0.0.1:8088:8088" because Docker tends to punch your firewall.